### ISSUE TITLE
Edit Template -> Edit a copy either results in empty control template or crashes designer

### ISSUE THREAD

**Issue by schnitty** [telerik/UI-For-UWP#99]
Within Blend if I want to edit a control template I can't do it through the designer.  I have to pull the generic.xaml for the relevant control from source and put into my ResourceDictionary.

**Comment by APopatanasov** [comment]
Hi @schnitty , 

We have tested several of of controls (RadDataGrid, RadDatePicker, RadRadialGauge) in Blend 2017 and all templates were extracted as expected. Please, provide some more details for which controls the described by you issue is observed.

**Comment by schnitty** [comment]
I just tried in Blend with 1.0.0.3 and the templates are extracting properly.  I have tried to extract the templates several times over the last few weeks and as recently as yesterday it wasn't working.  It would take a long time, not show the dialog that lets you select where you want to put it or would crash.  Maybe the issues I was having were related to the property panel crashing the designer.  Anyway happy that it works today with the 1.0.03 version!
